plugins/out_stackdriver/stackdriver_conf.c

```diff
@@ -42,12 +45,25 @@ static inline int key_cmp(const char *str, int len, const char *cmp) {
 
 static int validate_resource(const char *res)
 {
-    if (strcasecmp(res, "global") != 0 &&
-        strcasecmp(res, "gce_instance") != 0) {
-        return -1;
+    /*
+        * Resource types
+        * 'global', 'gce_instance', `generic_node', 'generic_task' are supported
```

Comment:
This just repeats the array below. How about just `Supported monitored resource types`?

Reply:
Done.